quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/FipsDistTest.java

```diff
@@ -105,6 +105,20 @@ void testHttpsBcfksTrustStoreInStrictMode(KeycloakDistribution dist) {
         });
     }
 
+    @Test
+    void testUnencryptedPkcs12TrustStoreInStrictMode(KeycloakDistribution dist) {
```

Comment:
Are there any implications around FIPS. PEM format works fine?

Reply:
The FIPS implication as I understand it are due to the additional password/encryption restrictions.  Since pem is unencrypted and does not use a password there are no issues there.

Reply:
@pedroigor Can you please confirm from the Core Team's perspective PEM is fine with FIPS?